recipes/strict-rfc3339/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "strict-rfc3339" %}
+{% set version = "0.7" %}
+{% set sha256 = "5cad17bedfc3af57b399db0fed32771f18fc54bbd917e85546088607ac5e1277" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - strict_rfc3339
+
+about:
+  home: http://www.danielrichman.co.uk/libraries/strict-rfc3339.html
+  license: GPLv3
```

Comment:
I think we prefer `GPL-3.0` following: https://opensource.org/licenses/alphabetical